.gitignore

```diff
@@ -3,3 +3,4 @@ config.yaml*
 store.json
 __debug_bin
 coverage.out
+.vscode/
```

Comment:
FYI: typically you should place this in your global .gitignore  (typically `~/.gitignore`)

Reply:
Ohh nice tip, I didn't know about `~/.gitignore` I will remove it I next commit.